common/app/navigation/helpers/UrlHelpers.scala

```diff
@@ -113,10 +117,30 @@ object UrlHelpers {
   def getContributionOrSupporterUrl(editionId: String)(implicit request: RequestHeader): String =
     if (editionId == "us") {
       getReaderRevenueUrl(Contribute, NewHeader)
+    } else if (editionId == "uk" && Switches.SupportFrontendActive.isSwitchedOn) {
+      getReaderRevenueUrl(Support, NewHeader)
     } else {
       getReaderRevenueUrl(Membership, NewHeader)
     }
 
+  def getSupportOrMembershipUrl(position: Position)(implicit request: RequestHeader): String = {
+    val editionId = Edition (request).id.toLowerCase()
```

Comment:
Nit: for consistency with the function below, could you remove the extra space after Edition